tests/common/config_reload.py

```diff
@@ -93,3 +95,7 @@ def config_reload(duthost, config_source='config_db', wait=120, start_bgp=True,
                     "PFC_WD is missing in CONFIG-DB")
     else:
         time.sleep(wait)
+
+    if check_intf_up_ports:
```

Comment:
Is there a case where it makes sense to use `check_intf_up_ports`, but `safe_reload` is not used? If not, it might be better to just have it within the `if safe_reload` block.

Reply:
@saiarcot895 - Do you mean to not have check_intf_up_ports at all, and do the interface up check when safe_reload is True. Or do you mean to have the check_intf_up_ports option, but do that checking only if safe_reload is set to true.